notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -163,7 +163,7 @@
       },
```

Comment:
<p>&nbsp;you will use -&gt; you use</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>